libs/template/config.go

```diff
@@ -8,29 +8,47 @@ import (
 
 	"github.com/databricks/cli/libs/cmdio"
 	"github.com/databricks/cli/libs/jsonschema"
+	"golang.org/x/exp/slices"
```

Comment:
since we're on version 1.21 you can just use `slices` now

Reply:
done